core/src/main/java/org/infinispan/security/actions/GetDefaultCacheConfigurationAction.java

```diff
@@ -0,0 +1,21 @@
+package org.infinispan.security.actions;
+
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.manager.EmbeddedCacheManager;
+
+/**
+ * GetDefaultCacheConfigurationAction.
+ *
+ * @since 14.0
+ */
+public class GetDefaultCacheConfigurationAction extends AbstractEmbeddedCacheManagerAction<Configuration> {
+
+   public GetDefaultCacheConfigurationAction(EmbeddedCacheManager cacheManager) {
+      super(cacheManager);
+   }
+
+   @Override
+   public Configuration run() {
+      return cacheManager.getDefaultCacheConfiguration();
```

Comment:
I forgot we still had that method around. It should have been deprecated. We should no longer have internals relying on knowledge about a "default" cache or its configuration.

Reply:
Updated